scripts/manila-share-snapshot.py

```diff
@@ -40,19 +41,23 @@ def do_GET(self):
         self.wfile.write(data.encode('utf-8'))
 
 
-class ManilaShareServerNanny(ManilaNanny):
-    """ Manila Share Server """
-    def __init__(self, config_file, interval, prom_port, http_port, handler):
-        super(ManilaShareServerNanny, self).__init__(config_file,
-                                                     interval,
-                                                     prom_port=prom_port,
-                                                     http_port=http_port,
-                                                     handler=handler)
+class ManilaShareSnapshotNanny(ManilaNanny):
+    """ Manila Share Snapshot """
+    def __init__(self, config_file, interval, tasks, dry_run_tasks, prom_port, http_port, handler):
+        super(ManilaShareSnapshotNanny, self).__init__(config_file,
+                                                       interval,
+                                                       prom_port=prom_port,
+                                                       http_port=http_port,
+                                                       handler=handler)
         self.orphan_snapshots_lock = Lock()
         self.orphan_snapshots: Dict[str, Dict[str, str]] = {}
         self.orphan_snapshots_gauge = Gauge('manila_nanny_orphan_share_snapshots',
                                             'Orphan Manila Share Snapshots',
                                             ['share_id', 'snapshot_id'])
+        self._task = tasks
+        self._task_dry_run = dry_run_tasks
+        if not any(tasks.values()):
+            raise Exception('All tasks are disabled')
```

Comment:
Raising here seems a bit hard as long as the orphan snapshots are not counted in as task.
But in the end I could live with it, since we anyhow would always enable one task, I believe